addons/test/misra/misra-test.c

```diff
@@ -68,9 +68,23 @@ int misra_5_2_field_hides_field1_31y;//5.2
 };
 const char *s41_1 = "\x41g"; // 4.1
 const char *s41_2 = "\x41\x42";
-const char *s41_3 = "\x41" "g";
+const char *s41_3 = "\x41" "\x42";
+const char *s41_4 = "\x41" "g";
+const char *s41_5 = "\x41\xA";
+const char *s41_6 = "\xA\x41";
+const char *s41_7 = "\xAA\xg\x41"; // 4.1
+const char *s41_8 = "\xAA\x\x41"; // 4.1
 int c41_3         = '\141t'; // 4.1
 int c41_4         = '\141\t';
+int c41_5         = '\0';
+int c41_6         = '\0\t';
+int c41_7         = '\12\t';
+int c41_8         = '\0t';   // 4.1
+int c41_9         = '\12';
+int c41_10        = '\12\n';
+int c41_11        = '\12n';  // 4.1
+int c41_12         = '\12323';
```

Comment:
IMHO this also violates the rule, isn't it similar to '\123t'?
I have read the rule several times, but I am not sure.
According to http://www.iso-9899.info/n1570.html an octal escape sequence looks like this:
```
             octal-escape-sequence:
                     \ octal-digit
                     \ octal-digit octal-digit
                     \ octal-digit octal-digit octal-digit
```
So the octal escape sequence `\123` in `'\12323'` is not terminated i would say.